BackendWebApp/src/test/resources/scripts/setUpDatabase.sh

```diff
@@ -0,0 +1,2 @@
+docker build -t biostudies-mysql ../;
+docker-compose -f ../docker-compose.yml up -d
```

Comment:
forse users to use `docker compose` may be an overkill  (and an extra dependency that need to be installed) only to get a mysql db  I would consider use `docker run -e` to provide environment variables.

Reply:
Done